lib/components/art/ArtDetailWidget.dart

```diff
@@ -1,5 +1,8 @@
+import 'package:artsideout_app/theme.dart';
 import 'package:flutter/material.dart';
 import 'package:artsideout_app/graphql/Installation.dart';
+import 'package:google_fonts/google_fonts.dart';
```

Comment:
We shouldn't use different fonts across different areas of the application. We should always be consistent so i think it would be a good idea for now and see what we can do for the application globally. This font is appealing so maybe try adding it in theme.dart and see if it looks good